client/src/components/LocalEmergencyBanner/LocalEmergencyBanner.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Icon } from "antd-mobile";
+import { theme } from "../../constants/theme";
+// import localEmergencyData from './localEmergencyData';
+import localEmergencyData from './emergency-numbers-by-code'
+import axios from "axios"
+
+const { typography, colors } = theme;
+
+const LocalEmergencyBannerStyle = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: #D1222C;
+  color: ${colors.white};
+  textAlign: ${typography.paragraph};
+  & .close {
+      align-self: flex-start;
+      & button {
+        position: relative;
+        padding: 0;
+        border: none;
+        background: transparent;
+        cursor: pointer;
+      }
+    }
+  .localEmergencyNumber {
+    flex: 1 0 auto;
+    font-size: ${typography.size};
+    justify-content: center;
+    align-self: center;
+    padding: 10px;
+  }
+  .emergencyNumberText{
+    font-size: 22px;
+  }
+  .emergencyNumber {
+    font-size: 22px;
+    font-weight: bold;
+  }
+`;
+
+const LocalEmergencyBanner = () => {
+  const [displayBanner, setDisplayBanner] = useState(true);
+  const [localEmergencyNumber, setLocalEmergencyNumber] = useState("")
+
+  // Shows banner until client closes
+  const bannerDisplay = () => {
+
+    if (!displayBanner){
+      return <div></div>;
+    } else {
+        axios.get('https://extreme-ip-lookup.com/json/')
+          .then(ipInfo => {
+            let countryCode = localEmergencyData[ipInfo.data.countryCode];
```

Comment:
```suggestion
            let countryCode = localEmergencyData[ipInfo.data.countryCode];
```

If for whatever reason we don't have a match (we should be just in case) , maybe we should throw an error so it displays 'location unavailable'?